tests/test_solvermode/test_speciestransport.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version("latest")
+def test_species_material(load_mixing_elbow_mesh):
```

Comment:
Maybe name it `test_create_mixture`?

Reply:
What if we called it `test_change_create_mixture` since I'm also editing, copying, and deleting objects?

Reply:
I would say `test_species_materials` (added `s` at the end) is slightly better, as you are testing `setup.species` and `setup.materials` (`mixture` is under `materials` as well). 

Alternatively, if you would like to compete for the longest test names with Mainak, a very detailed `test_species_materials_mixture_create_change_copy_delete` would be great, as these are all the operations you are testing here

But any one of these suggested sound OK to me

Reply:
It's true that the test is asserting a number of things but for me the main point of it is to illustrate how to create a new mixture from scratch.  For me a longer name is a bit cluttered and `test_species_materials` does not seem accurate enough.

Reply:
I think that moving the creation part into a function and breaking this down into two or three separate tests might be a little excessive, but could work as well.

And I agree, I previously meant that the suggested `test_create_mixture` or `test_change_create_mixture` sound fine to me too